### PR TITLE
Fixed panic when display error that wraps error that has expired hidd…

### PR DESCRIPTION
**Pull request by oxtoacart**
…en cause

For getlantern/lantern-internal#1428

### CODE REVIEW

**Comment by oxtoacart** [comment]
@joesis @myleshorton This is a subtle one. The root cause is that a [typed interface value is not nil](https://play.golang.org/p/Ergz3qIO-s), which messes up some of the nil checks that we're using in our printing logic.

File: errors_test.go [hunk 1]
```diff
@@ -117,3 +117,24 @@ func TestWrapNil(t *testing.T) {
 func doWrapNil() error {
 	return Wrap(nil)
 }
+
+func TestHiddenWithCause(t *testing.T) {
```
Comment (oxtoacart) [inline errors_test.go]: Before the fix, this would panic.

**Comment by myleshorton** [comment]
LGTM @oxtoacart!

**Comment by oxtoacart** [comment]
Thanks @myleshorton!

**Pull request merged**